src/styles/main.scss

```
@@ -1,3 +1,77 @@
+$cellHeight: 100px;
+$calendarGap: 1px;
+$calendarPadding: 10px;
+$calendarMaxWidth: 7 * $cellHeight + 6 * $calendarGap + 2 * $calendarPadding;
+$c-black: #000;
+$c-white: #eee;
+$c-pink: pink;
+$transitionDuraiton: 0.5s;
+$daysInWeek:
+  "mon" 0,
+  "tue" $cellHeight + $calendarGap,
+  "wed" 2 * $cellHeight + $calendarGap,
+  "thu" 3 * $cellHeight + 2 * $calendarGap,
+  "fri" 4 * $cellHeight + 3 * $calendarGap,
+  "sat" 5 * $cellHeight + 4 * $calendarGap,
+  "sun" 6 * $cellHeight + 5 * $calendarGap;
+
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
```

```suggestion
body {
  margin: 0;
  height: 100vh;
  display: flex;
}
```
its makes calendare in the middle of the screen vertically